tests/test_day_at_the_supermarket.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+import pytest
+
+from codecademy.day_at_the_supermarket import compute_bill
+
+
+@pytest.mark.parametrize("test_compute_bill_input,test_compute_bill_expected", [
```

Comment:
your naming of the test parameters is too explicit:
`test_compute_bill_input` could be just `input` and `test_compute_bill_expected` >> `expected`
because these are the parameters applied only to `test_compute_bill()` test function. This is how it should be:
```python
def test_compute_bill(input, expected):
    assert compute_bill(input) == expected
```
Now it's A LOT easier to read.
Here the only meaningful name that needs to be is the name of the test function, which you already did - `test_compute_bill`